docs/process/releasing-updates.md

```diff
@@ -6,25 +6,60 @@ We have three channels to which we can release: `production`, `beta`, and `test`
 
 - `production` is the channel from which the general public downloads and receives updates. It should be stable and polished.
 
-- `beta` is released more often than `production`. It may be buggy and unpolished.
+- `beta` is released more often than `production`. We want to ensure `master` is always in a state where it can be released to users and we will use `beta` for additional QA.
 
 - `test` is unlike the other two. It does not receive updates. Each test release is locked in time. It's used entirely for providing test releases.
 
 ## The Process
 
-From a clean working directory, set the `GITHUB_ACCESS_TOKEN` environment variable to a valid [Personal Access Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) and run:
+### 1. GitHub Access Token
+
+From a clean working directory, set the `GITHUB_ACCESS_TOKEN` environment variable to a valid [Personal Access Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) 
+
+You can check that this is set on macOS by:
+```shellsession
+$ env GITHUB_ACCESS_TOKEN
+```
+
+You can check that this is set on Windows by:
+```shellsession
+$ echo %GITHUB_ACCESS_TOKEN%
+```
+
+If you are creating a new Personal Access Token on GitHub:
+* make the token memorable - use a description like `Desktop Draft Release and Changelog Generator`
+* the `read:org` scope is the **only** required scope for drafting releases
+
+You can add an access token as an environment variable on macOS by: 
+```shellsession
+$ export GITHUB_ACCESS_TOKEN={your token here}
+```
+
+You can add an access token as an environment variable on Windows by: 
+```shellsession
+$ set GITHUB_ACCESS_TOKEN={your token here}
+```
+
+### 2. Create Draft Release
+
+Once the personal access token is set, run the script below, which will determine the next version from what was previously published, based on the desired channel.
+
+For `production` and `beta` releases, run:
 
 ```shellsession
 $ yarn draft-release (production|beta)
 ```
 
-This command will then explain the next steps:
+(For `test` releases, follow the directions in the steps below to update `app/package.json`'s `version` to a higher version and add a changelog entry. The script does not support test yet.)
+
+The script will output a draft changelog, which covers everything that's been merged, and probably needs some love.
```

Comment:
:art: a bit more detail

> which **lists** everything that's been merged **since the last release**. **This is a generated list, so it should only be used as a starting point.**